packages/frontend/src/components/login/WelcomeBackgroundList.tsx

```diff
@@ -24,12 +24,15 @@ const WelcomeBackgroundList: React.FC<ScrollingModalProps> = ({
         const scrollPosition = container.scrollTop
 
         children.forEach((child) => {
-            const childTop = child.offsetTop - scrollPosition + 50
-            const childBottom = childTop + child.clientHeight - 50
+            const childTop = child.offsetTop - scrollPosition
+            const childBottom = childTop + child.clientHeight
+            const middle = containerHeight / 2
 
-            const middle = (containerHeight - child.clientHeight + 200) / 2
+            const centerZoneHeight = 200
+            const centerTop = middle - centerZoneHeight / 2 + 100
+            const centerBottom = middle + centerZoneHeight / 2 - 50
```

Comment:
These are not related to `child`, so could be moved outside the loop

Reply:
also the `centerZoneHeight` might be changed on **mobile**?
Since now the mobile version would look like this:
<img width="401" alt="image" src="https://github.com/social-tw/social-tw-website/assets/9023842/80c89646-88dc-46c9-9f1c-e873a9a7493f">

Reply:
已更改